html-forms/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Html Forms</title>
+  </head>
+  <body>
+    <form action="index.html" method="get">
+      <h1>Coding Form!</h1>
+      <div>
+        <label>Email: <br /></label>
+        <input type="text" name="Email" />
+      </div>
+      <div>
+        <label>Why is coding cool? <br /></label>
+        <select name="Select an option">
+          <option value="Select an option" selected="selected">
+            Select an option
+          </option>
+          <option value="Because it's interesting">
+            Because it's interesting
+          </option>
+          <option value="Because it's fun">Because it's fun</option>
+          <option value="Because it's powerful">Because it's powerful</option>
+        </select>
+      </div>
+      <div>
+        <label>Favorite Language (choose one) <br /></label>
+        <input type="radio" name="language" value="HTML" />HTML
+        <input type="radio" name="language" value="CSS" />CSS
+        <input type="radio" name="language" value="JavaScript" />JavaScript
+      </div>
+      <div>
+        <label>Would you like to sign up for more information? <br /></label>
+        <input type="checkbox" name="signup" value="Sign me up!" /> Sign me up!
+      </div>
+      <div>
+        <label>Comments <br /></label>
```

Comment:
This `label` needs to be configured with a `for` attribute that matches the `id` attribute on its form control element.